html/log.php

```diff
@@ -0,0 +1,98 @@
+<?php

```

Comment:
I don't really see why the PHP parsing is necessary at all. I think we should scrap the PHP and just parse the log file all in Python. This just adds an extra layer of complexity.

Reply:
Probably a good idea, will do that then (as soon as i can)